scalecube-organization/src/main/java/io/scalecube/organization/tokens/store/ApiKeyBuilder.java

```diff
@@ -38,17 +39,15 @@ public static ApiKey build(
       tokenClaims.put(ROLE_KEY, Role.Member.toString());
     }
 
-    PrivateKey signingKey = keyStore.getPrivateKey(organization.keyId());
-
     return JwtApiKey.builder()
         .issuer(ISSUER)
-        .subject(organization.id())
+        .subject(orgId)
         .name(request.apiKeyName())
         .claims(tokenClaims)
-        .id(organization.id())
-        .audience(organization.id())
+        .id(orgId)
+        .audience(orgId)
         .expiration(tokenExpiration.value().orElse(null))
-        .build(organization.keyId(), signingKey);
+        .build(keyId, signingKey);
```

Comment:
what is the idea of having paramters in the build ?
why not just .build()?

Reply:
Changed. Done.